python/test/layer_harness.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+import shutil
+import numpy as np
+
+import os
+
+from neuroglancer.pipeline.storage import Storage
+from neuroglancer.pipeline.task_creation import (upload_build_chunks, create_info_file_from_build,
+    create_ingest_task, MockTaskQueue)
+
+layer_path = '/tmp/removeme/'
+
+def create_storage(layer_name='layer'):
+    uri = 'file://' + os.path.join(layer_path, layer_name)
+    return Storage(uri, n_threads=0)
```

Comment:
I'm worried about n_threads=0, I think it's likely that Storage client will use the default value which uses threads so we might want to use that value, so threading get tested